submissions/dashakim/oop-task/js/app.js

```diff
@@ -0,0 +1,100 @@
+//game score
+var score = 0
+//Enemy class with update(), render() methods
+var Enemy = function (x, y, speed) {
+    this.x = x
+    this.y = y
+    this.speed = speed
+    this.sprite = 'images/car.png';
```

Comment:
There are disputes on whether to use semicolons in Javascript or to skip those. But the certain style should be followed across the entire codebase. Please choose your side and fix semicolons.